cli/code_scanner.py

```diff
@@ -423,6 +423,7 @@ def poll_scan_results(context: click.Context, cycode_client, scan_id: str, polli
     while time.time() < end_polling_time:
         scan_details = cycode_client.get_scan_details(scan_id)
         if scan_details.scan_update_at is not None and scan_details.scan_update_at != last_scan_update_at:
+            click.echo()
```

Comment:
?

Reply:
 When halo spinner is running and we need to print a new message from server than we need to get a new refresh line in order to prevent it to be on same line

Reply:
please don't forget what we discussed yesterday - 
if you want to print all the progress logs -> use the logger instead of Halo
if you want to display the latest progress only -> remove this line

Reply:
@morsa4406 Done?

Reply:
Yes 👍  thx